conda-forge.yml

```diff
@@ -1,9 +1,20 @@
-compiler_stack: comp7
-max_py_ver: '37'
-max_r_ver: '35'
+azure:
+  settings_win:
+    pool:
+      vmImage: windows-2019
+build_platform:
+  osx_arm64: osx_64
+  linux_ppc64le: linux_64
+  linux_aarch64: linux_64
+os_version:
+  linux_64: cos7
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  win: azure
+  linux_aarch64: azure
+  linux_ppc64le: azure
```

Comment:
This is redundant with specifying the build platform (which for `linux-64` is necessarily on azure, and on win it's the default)
```suggestion
```